gems/quilt_rails/README.md

```diff
@@ -186,9 +392,16 @@ function App() {
 export default App;
 ```
 
+### Isomorphic state
+
+With SSR enabled React apps, state must be serialized on the server and deserialized on the client to keep it consistent. When using `@shopify/react-server`, the best tool for this job is [`@shopify/react-html`](https://github.com/Shopify/quilt/tree/master/packages/react-html)'s [`useSerialized`](https://github.com/Shopify/quilt/tree/master/packages/react-html#in-your-application-code) hook.
```

Comment:
I think this section is still pretty impossible to understand as an outsider so we probably want to add some real examples here. We may want to have a `recipes` section or something like that in the future.